docs/content/management.md

```diff
@@ -626,6 +632,21 @@ the following additional fields.
 Services should reply with HTTP status `200 OK` if the status update is
 processed successfully.
 
+### Local Status Logs
+
+Local console logging of status updates can be enabled via the `console` config option.
+This does not require any remote server. Example of minimal config to enable:
+
+```yaml
+status:
+    console: true
+```
+This will dump all status updates through the OPA logging system at the `info` level. See
+[Configuration Reference](../configuration) for more details.
+
+> Warning: this can be very verbose! The `metrics.prometheus` portion of the status update
```

Comment:
Perhaps it would be good to mention that while these log messages are verbose, they are relatively infrequent.

Reply:
👍 Good call.